### PR TITLE
Added xlsx and csv support

### PR DESCRIPTION
**Pull request by dipta10**
Issue: https://github.com/grumpyp/aixplora/issues/27

- For xlsx file, I'm converting this to a CSV file and then saving it.
- For csv file, I'm saving this as it is, just as a text file. This seems to work. (and to be honest, I'm not sure how else to represent the file 😅)

Sample xlsx file: 
[Book.xlsx](https://github.com/grumpyp/aixplora/files/11583331/Book.xlsx)

Sample result:
Screenshot 2023-05-28 064441

### CODE REVIEW

File: backend/database/database.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 from sqlalchemy import create_engine, text
-from sqlalchemy.ext.declarative import declarative_base
```
Comment (grumpyp) [inline backend/database/database.py:2]: Why do you delete that?

File: backend/loaders/xlsx_loader.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    workbook = openpyxl.load_workbook(excel_file)
+
+    # Open the CSV file for writing
+    output_file_path = f"{misc_dir}/{filename}.txt"
+    with open(output_file_path, "w") as csvfile:
+        writer = csv.writer(csvfile)
+
+        # Iterate over all worksheets
+        for sheet_name in workbook.sheetnames:
+            writer.writerow([f'Sheet name: {sheet_name}'])
```
Comment (grumpyp) [inline backend/loaders/xlsx_loader.py:23]: I believe adding this makes no sense, if you want to add sheet meta data, pls add it to meta data. 

We're now "confusing" the embeddings with that information, see logs:

```
Sheet name: Tabelle1\n,,,,,\n,,,,,\n,,,,,\n,,,,,\n,,,,,
```

File: backend/utils.py [hunk 3]
```diff
@@ -17,6 +18,8 @@
     ".webm": lambda file: Whisperexporter().whisper_to_text(file.file, filename=file.filename, file_meta=file),
     ".m4a": lambda file: Whisperexporter().whisper_to_text(file.file, filename=file.filename, file_meta=file),
     ".docx": lambda file: load_docx(file.file, filename=file.filename, file_meta=file),
+    ".xlsx": lambda file: load_xlsx(file.file, filename=file.filename, file_meta=file),
+    ".csv": lambda file: load_txt(file.file, filename=file.filename, file_meta=file),
```
Comment (grumpyp) [inline backend/utils.py:22]: Please test if `.xls` also works

**Review by grumpyp — COMMENTED** [review]
nice !

File: backend/database/database.py (hunk 1 above)
Comment (dipta10) [reply backend/database/database.py:2]: It was an un-used import. do we still need this?

File: backend/utils.py (hunk 3 above)
Comment (dipta10) [reply backend/utils.py:22]: No, it throws error with `.xls` file. I think we'll need to use [xlrd](https://xlrd.readthedocs.io/en/latest/) to work with `.xls` file.

Comment (grumpyp) [reply backend/utils.py:22]: Would xlrd work with all kinds? Then I suggest using it

**Review by grumpyp — CHANGES_REQUESTED** [review]
if xlrd works with `.xls` too, I suggest using it instead of openpyxl

File: backend/utils.py (hunk 3 above)
Comment (dipta10) [reply backend/utils.py:22]: Sadly `xlrd` only works with `xls`
> This library will no longer read anything other than .xls files. For alternatives that read newer file formats, please see http://www.python-excel.org/

And [openpyxl](https://openpyxl.readthedocs.io/en/stable/) only works with `xlsx` files.
Could I address `xls` file in another issue/PR then?

**Comment by grumpyp** [comment]
merged in #31

Thanks a lot!!

**Pull request closed without merging**